ocs_ci/helpers/helpers.py

```diff
@@ -984,7 +984,7 @@ def validate_cephfilesystem(fs_name):
 
 
 def create_ocs_object_from_kind_and_name(
-    kind, resource_name, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+    kind, resource_name, namespace=config.ENV_DATA["cluster_namespace"]
```

Comment:
I'm not sure, if it might be problem or not in this case, but using value from dictionary key here means, that the default `namespace` value will be the one present in the `config` in the time of initialization of the function, not the value from `config` from the time when the function will be executed (I'm not sure, how to describe it better).
In other words, if this function will be initialized before whole configuration will be loaded (or if the `config.ENV_DATA["cluster_namespace"]` will be changed later for some other reason, which is probably not the case here), the default value for `namespace` parameter will be wrong.

Reply:
It seems that for deployment and regression runs is used correct value according to verification runs. I believe that it should contain correct value at the time of the execution but please let me know if you think that this needs to be changed. I can keep the argument as it is and change it in function calls.

Reply:
I was thinking about setting the default `namespace` to `None` and then in the beginning of the function set the actual default value:
```
if `namespace` is none`:
  namespace = config.ENV_DATA["cluster_namespace"]
```

But it is maybe over-complicated and not necessary in this case.

Reply:
Sounds good to me. I will make the change.

Reply:
There are other similar cases (setting default value for function), so I would propose to change it there also.